src/NServiceBus.RabbitMQ/Connection/ConnectionFactory.cs

```diff
@@ -46,11 +47,18 @@ public ConnectionFactory(ConnectionConfiguration connectionConfiguration)
             }
         }
 
+        public IConnection CreatePublishConnection() => CreateConnection("Publish");
+
+        public IConnection CreateAdministrationConnection() => CreateConnection("Administration");
+
         public IConnection CreateConnection(string connectionName)
         {
-            connectionFactory.ClientProperties["connected"] = DateTime.Now.ToString("G");
+            lock (lockObject)
```

Comment:
I could make this change, but I tend to prefer separate lock objects. You never know when some code is going to do something dumb like `lock (this)` and deadlock themselves.